lib/companies_house/client.rb

```diff
@@ -49,6 +50,14 @@ def officers(id)
       items
     end
 
+    def company_search(query, per_page: nil, offset: nil)
```

Comment:
Could we make these parameter names the same as the names in the API? It's probably nicer that way. (Sorry for another suggestion!)

Reply:
I thought about how we should name these and went with the GoCardless API naming convention. Do you think following the CH convention makes more sense here?

Reply:
I'd say that the CH convention makes sense. I think you'd be looking at the Companies House API documentation, and you'd expect the gem to match it fairly closely in how it works - particularly for things like parameter names.

Reply:
Cool, updated.